public/static/docs/command-reference/push.md

```diff
@@ -244,6 +250,11 @@ $ tree ../vault/recursive
 5 directories, 5 files
 ```
 
+> Cache files with `.dir` extension is a special text file that contains the
```

Comment:
same as above

Reply:
OK just left link to /doc/user-guide/dvc-files-and-directories#structure-of-cache-directory

See bf82ff22.